test/box-luatest/gh_7149_forbid_ddl_until_box_schema_upgrade_test.lua

```diff
@@ -35,8 +35,8 @@ g.test_ddl_ops = function()
     g.server:exec(function()
         -- DDL with old schema is forbidden.
         local errmsg = 'Your schema version is 2.10.4 while Tarantool ' ..
-                       box.info.version ..
-                       ' requires a more recent schema version. ' ..
+                       'feature SCHEMA_FEATURE_DDL_BEFORE_UPGRADE ' ..
```

Comment:
The feature name looks rather ugly here. I'd prefer something like this instead:

You schema version is *2.10.4* while to use *DDL without schema upgrade*, schema version *2.11.1* or higher is required. Please ,consider using box.schema.upgrade().

Reply:
Done